### DISCUSSION TITLE
How to debug the LSP not starting?

### DISCUSSION THREAD

**Discussion by Aceticia** [LunarVim/LunarVim#2662] [Q&A] [answered]
I'm on MacOS and running nvim 0.7.

Two of the LSPs are not working for me: ltex-ls (for latex) and sourcery (for python). Other language servers of these languages work fine: texlab (latex), pyright (python). Calling `LspInfo` doesn't show any information about ltex-ls and sourcery, but others like null-ls and pyright are listed and automatically started. Reinstalling them from within LspInstaller doesn't work, and manually removing the directory they were installed in, then installing again, doesn't work either. 

I don't know if this is expected or not, but these two particular servers does not seem to be in my PATH. I doubt this will make a difference though, since pyright doesn't seem to be on the PATH but still is able to run.

I previously used `sourcery` as a python LSP when not using lvim, and it worked well just out of the box with lsp-installer, so I'm suspecting this problem came from within lvim? I'm not quite sure where to post this, so please let me know if I should post in other places like lsp-config or lsp-installer.

**Comment by LostInTheLogs** [comment, marked as answer]
check logs using `leader L l L`
